docs/assets/js/swagger.yml

```
@@ -89,14 +77,18 @@ components:
       properties:
         has_more:
           type: boolean
+          description: Next page is available
         next_offset:
           type: string
+          description: Token used to retrieve the next page
         results:
           type: integer
           minimum: 0
+          description: The number of values found in the results
```

nit:
```suggestion
          description: Number of values found in the results
```

But I think this is *not* the total number of values, but somehow the number of entries returned on this page?